src/platforms/web/util/model.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+
+export function getModelModifier (vnode: VNodeWithData): ASTModifiers | void {
+  const directives = vnode.data.directives || []
+  for (let i = 0, directive; i < directives.length; i++) {
+    directive = directives[i]
+    if (directive.name === 'model') {
+      return directive.modifiers
+    }
+  }
+}
```

Comment:
I think that this function is not problem, however, If modifier did not find, in end of function, 
when you explicitly return, we can easily understand about this function.